app/views/DR/Maps/Map.js

```diff
@@ -154,7 +147,9 @@ export default function HospitalMap({ route: { name: type } }) {
         <Icon name='thermometer-quarter' size={22} color='#4372e8' />
       )}
       <Text style={styles.cardTitle}>
-        {type === 'Hospitals' ? 'Hospitales' : 'Laboratorios'}
+        {type === 'Hospitals'
+          ? languages.t('navigation.hospitals_maps')
```

Comment:
If you apply the previous changes you can delete languages from here and the other one
```suggestion
          ? t('navigation.hospitals_maps')
```